src/libs/Navigation/AppNavigator/Navigators/BottomTabNavigator.tsx

```diff
@@ -0,0 +1,44 @@
+import {useNavigationState} from '@react-navigation/native';
+import type {StackNavigationOptions} from '@react-navigation/stack';
+import React from 'react';
+import createCustomBottomTabNavigator from '@libs/Navigation/AppNavigator/createCustomBottomTabNavigator';
+import getTopmostCentralPaneRoute from '@libs/Navigation/getTopmostCentralPaneRoute';
+import type {BottomTabNavigatorParamList} from '@libs/Navigation/types';
+import AllSettingsScreen from '@pages/home/sidebar/AllSettingsScreen';
+import SidebarScreen from '@pages/home/sidebar/SidebarScreen';
+import SCREENS from '@src/SCREENS';
+import ActiveRouteContext from './ActiveRouteContext';
+
+const loadWorkspaceInitialPage = () => require('../../../../pages/workspace/WorkspaceInitialPage').default as React.ComponentType;
+
+const Tab = createCustomBottomTabNavigator<BottomTabNavigatorParamList>();
+
+const screenOptions: StackNavigationOptions = {
+    headerShown: false,
+};
+
+function BottomTabNavigator() {
+    const activeRoute = useNavigationState(getTopmostCentralPaneRoute);
```

Comment:
We tried to handle it this way, but to obtain information about activeRoute we need to call this hook inside the parent route because that's where this info is stored